SharedPackages/BrowserServicesKit/Sources/PixelExperimentKitTestUtils/MockExperimentActionPixelStore.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  MockExperimentActionPixelStore.swift
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import PixelExperimentKit
+
+public class MockExperimentActionPixelStore: ExperimentActionPixelStore {
```

Comment:
Please refer to this comment for more information on this change: https://github.com/duckduckgo/apple-browsers/pull/562/files#r2055883698

I needed to make these mocks available to the main app.

Reply:
I left this in for now, we can opt to change it in the future but it should be pretty harmless to keep this way for now.